packages/react-sdk/src/state/crdt/documents/elements.ts

```diff
@@ -114,7 +114,10 @@ export type ImageElement = ElementBase & {
    */
   mxc: string;
   fileName: string;
-  mimeType: ImageMimeType;
+  /**
+   * @deprecated  This is kept for backwards compatibility. We dont send it anymore. DO NOT USE!
```

Comment:
Maybe also mention it [here](https://github.com/nordeck/matrix-neoboard/blob/fd631c6b9b92bf7d04444e1a4e72c2ccd99e767f/docs/model/crdt-documents.md?plain=1#L195).